lib/bin/supllib/include/supl_os_client.h

```diff
@@ -0,0 +1,25 @@
+#include <nrf_socket.h>
+#include <supl_session.h>
+
+/**
+ * @brief Run this function to start a SUPL session
+ *
+ * @param[in] agps_request This contain the information about the AGPS data
+ *                         that the GPS module is requesting from the server.
+ *
+ * @return 0  SUPL session was successful.
+ *         <0 SUPL session failed.
+ */
+int nrf_supl_session(nrf_gnss_agps_data_frame_t *agps_request);
```

Comment:
Couldn't this be simply `supl_session` and `supl_init`? We tend to avoid `nrf_` prefixing in here.

Reply:
Yes, I agree, but the library functions are called that so this would lead to name crash. I can of course rename the functions in the library to something else to free up these names. You thoughts?

Reply:
The header file is `supl_os_client.h` so perhaps `supl_client` would be a good prefix for these functions?

Reply:
The library is now exposing supl_client_init, supl_client_session and supl_client_abort. So renamed the functions without the nrf_ prefix.